lms/djangoapps/course_home_api/urls.py

```diff
@@ -59,7 +59,14 @@
 # Progress Tab URLs
 urlpatterns += [
     re_path(
-        fr'v1/progress/{settings.COURSE_KEY_PATTERN}',
+        fr'v1/progress/{settings.COURSE_KEY_PATTERN}$',
```

Comment:
Does this mean that you can't add a trailing slash? And in the next pattern, does it mean you have to add a trailing slash?

I think it'd make sense if these two at least agreed on a slash or not. Ideally could accept either? But that seems less important.

Reply:
(that's fine - but does not having a termination symbole create other problems? Like your impulse was good, presumably to prevent users throwing something random at end? Or are these url patterns handled smartly by django, for us, such that it does termination logic like `/?$` for us?)

Reply:
i had initially done that mostly to simplify determining which of the two views we were in, but i just adjusted the urls file and tested so that all of these urls now work properly
`http://localhost:18000/api/course_home/v1/progress/course-v1:edX+DemoX+Demo_Course
http://localhost:18000/api/course_home/v1/progress/course-v1:edX+DemoX+Demo_Course/
http://localhost:18000/api/course_home/v1/progress/course-v1:edX+DemoX+Demo_Course/1
http://localhost:18000/api/course_home/v1/progress/course-v1:edX+DemoX+Demo_Course/1/`